src/services/schema-export/utils.ts

```diff
@@ -7,8 +9,38 @@ export function isPrimitive(type: string | undefined) {
     case 'string':
     case 'data':
     case 'date':
+    case 'object id':
+    case 'objectId':
+    case 'decimal':
+    case 'decimal128':
       return true;
     default:
       return false;
   }
 }
+
+export const reMapType = (type: string): string => {
```

Comment:
I think this needs a comment :) And perhaps a (longer) more descriptive name.

Reply:
Yes.. it was a surprise that this kind of mapping was still needed.
Will add a comment or a different solution.

Reply:
The mapping will be obsolete once https://github.com/realm/realm-js/pull/3235 is merged & consumed.
I've added todos to remove these temp fixes.